src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -47,15 +80,8 @@ public void render(float delta) {
         camera.update();
         batch.setProjectionMatrix(camera.combined);
 
-        batch.begin(ShapeRenderer.ShapeType.Filled);
```

Comment:
Pretty sure you want to keep this line of code, to minimize merge conflicts as we probably want to use it in the master.